cowprotocol/cowswap
# [1] fix(twap): display fallback handler banner when it's relevant

# Summary

| Not in Safe | In Safe | In Safe + checked |
|--------|--------|--------|
| <img width="506" alt="Screenshot 2023-06-30 at 19 31 55" src="https://github.com/cowprotocol/cowswap/assets/7122625/c9ce41d6-a6ba-47e7-8354-41230d538415"> | <img width="501" alt="Screenshot 2023-06-30 at 19 32 45" src="https://github.com/cowprotocol/cowswap/assets/7122625/e6e8dd67-f821-4745-b732-f8219a2f81de"> | <img width="502" alt="Screenshot 2023-06-30 at 19 32 50" src="https://github.com/cowprotocol/cowswap/assets/7122625/0f44419e-91fb-4d9f-96d6-6d1ae17eb2f3"> | 

1. Removed state "Unsupported Safe"
2. Display checkbox "Modify safe" fallback handler is not set and approval is needed
3. Added a line break for "Unsupported wallet detected" banner

  # To Test

1. Open Advanced orders NOT in Safe
- [ ] The button is disabled
- [ ] "Unsupported Wallet detected" banned is displayed
2. Open Advanced orders in Safe without Fallback handler
- [ ] The button is disabled
- [ ] "Unsupported Safe detected" banned is displayed
3. "Modify Safe's fallback handler" checkbox works well
